Extension/main.css

```diff
@@ -38,6 +42,12 @@ li:hover {
     background: url('email16.png') no-repeat 5px 3px transparent;
 }
 
+.html_link {
+    background-color: white;

```

Comment:
When copying the HTML, we get all the cascaded style sheets with it. So, I chose the default font/font-size and bg color of Outlook.

Reply:
It's a little more complicated than that. If I paste on a blue background in Outlook, the background _stays blue_.